src/libraries/Microsoft.PowerFx.Core/Types/Enums/ArgPreprocessor.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Core.Types.Enums
```

Comment:
>Microsoft.PowerFx.Core.Types.Enums [](http://example.com/codeflow?start=10&length=34)

super nit - I think this namespace is for our publicly supported language enums; not the internal enums we use in our code. 

Maybe move next to TexlFunction?

Reply:
updated.